lua/nvim-web-devicons/icons-default.lua

```diff
@@ -998,6 +1028,12 @@ local icons_by_file_extension = {
     cterm_color = "251",
     name = "Crystal",
   },
+  ["crdownload"] = {
+    icon = "󰇚",
```

Comment:
Could we please avoid material design icons?

How about nf-fa-download  for all of the download types?

Reply:
I forget about that, changed. 55cad63f4d77